src/main/java/org/broadinstitute/hellbender/tools/walkers/genotyper/GenotypingEngine.java

```diff
@@ -260,30 +264,23 @@ static boolean noAllelesOrFirstAlleleIsNotNonRef(List<Allele> altAlleles) {
     protected abstract String callSourceString();
 
     /**
-     * Holds information about the alternative allele subsetting based on supporting evidence, genotyping and
-     * output modes.
-     */
-    private static class OutputAlleleSubset {
-        private  final List<Allele> alleles;
-        private  final boolean siteIsMonomorphic;
```

Comment:
i know this is wrong and some ancient detritus... don't open this can of worms here... its not worth the trouble and we have to do this quick

Reply:
This is a completely automated refactor so it just saves a few lines of code and does the same thing, but I can revert it.